tap_notion/client.py

```diff
@@ -81,7 +81,7 @@ def headers(self) -> Dict[str, str]:
     def authenticate(self, headers: Dict, params: Dict) -> Tuple[Dict, Dict]:
         """Injects authorization + Notion version headers"""
         headers["Authorization"] = f"Bearer {self.config['auth_token']}"
-        headers["Notion-Version"] = "2022-06-28"
+        headers["Notion-Version"] = "2025-09-03"
```

Comment:
Inconsistent 'Notion-Version's used in the code i.e. "2025-09-03" and "2022-06-28".